eng/common/scripts/Update-DocsMsMetadata.ps1

```diff
@@ -102,7 +114,18 @@ function GetAdjustedReadmeContent($ReadmeContent, $PackageInfo, $PackageMetadata
   if ($codeOwnerArray) {
     Write-Host "Code Owners are $($codeOwnerArray -join ",")"
     $author = $codeOwnerArray[0]
-    $msauthor = $author # This is a placeholder for now. Will change to the right ms alias.
+    try {
+      $msauthor = ."$PSScriptRoot/Get-AADIdentityFromGithubUser.ps1" `
+                    -TenantId $TenantId `
+                    -ClientId $ClientId `
+                    -ClientSecret $ClientSecret `
+                    -GithubUser $author
+    }
+    catch {
```

Comment:
Will the script ever throw an exception?

Reply:
No. I should check $LastExitCode